deploy.sh

```diff
@@ -0,0 +1,3 @@
+rm -rf build/ dist genyrator.egg-info/
```

Comment:
If this needs to be run with bash could we add a shebang line to the start? `#!/usr/bin/env bash`